### PR TITLE
Accounts api

### PR DESCRIPTION
**Pull request by liyocee**
Account API Client for interacting with the accounts API.

### CODE REVIEW

**Comment by liyocee** [comment]
@KayLuke , this is ready for review.

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def read(*rnames):
     import shutil
     shutil.copyfile('README.md', 'README.txt')
 
-VERSION = "0.1.7"
+VERSION = "0.2.7"
```
Comment (KayLuke) [inline setup.py]: Next version should be 0.1.8 (Always increment from the left)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Changelog
+============
+
+v0.2.7
```
Comment (KayLuke) [inline CHANGELOG.md]: 0.1.8

File: setup.py (hunk 1 above)
Comment (liyocee) [inline setup.py]: Great. Are we not using semantic versioning? http://semver.org/

**Pull request merged**